### DISCUSSION TITLE
[Feature] 360 pics on Android App

### DISCUSSION THREAD

**Discussion by divadwong68** [immich-app/immich#6903] [Feature Request]
### The feature

Is there any plans to show 360 pictures on Android phones?   I've uploaded pics on to Immich and it shows correctly on the Web but the Android app shows the picture flattened. 

Thanks

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of  #6572
